docs/build/guides/transactions/send-and-receive-payments.mdx

```diff
@@ -6,1405 +6,574 @@ sidebar_position: 7
-const sendTransactionResponse = await rpc.sendTransaction(transaction);
-
-// Here we check the status of the transaction as there are
-// a possible outcomes after sending a transaction that would have
-// to be handled accordingly, such as "DUPLICATE" or "TRY_AGAIN_LATER".
-if (sendTransactionResponse.status !== "PENDING") {
-  throw new Error(
-    `Failed to send transaction, status: ${sendTransactionResponse.status}`
-  );
-}
-
-// Here we poll the transaction status to await for its final result.
-// We can use the transaction hash to poll the transaction status later.
-const finalStatus = await rpc.pollTransaction(sendTransactionResponse.hash, {
-  sleepStrategy: (_iter: number) => 500,
-  attempts: 5,
-});
-
-// The pollTransaction method will return the final status of the transaction
-// after the specified number of attempts or when the transaction is finalized.
-// We then check the final status of the transaction and handle it accordingly.
-switch (finalStatus.status) {
-  case StellarSdk.rpc.Api.GetTransactionStatus.FAILED:
-  case StellarSdk.rpc.Api.GetTransactionStatus.NOT_FOUND:
-    throw new Error(`Transaction failed with status: ${finalStatus.status}`);
-  case StellarSdk.rpc.Api.GetTransactionStatus.SUCCESS:
-    console.log("Success! Results:", finalStatus);
-    break;
-}
-
-```
-
-</CodeExample>
-
-The key differences when using RPC instead of Horizon:
-
-1. **Instant submission**: `rpc.sendTransaction()` returns immediately with a "PENDING" status
-2. **Active polling required**: You must poll `rpc.pollTransaction()` to get the final result
-3. **Polling configuration**: You can customize polling intervals and retry attempts
-4. **Status handling**: Check for SUCCESS, FAILED, or NOT_FOUND status in the final result
-
-#### Using the Horizon API
-
-<CodeExample>
-
-```js
-import * as StellarSdk from "stellar-sdk";
-
-// Initialize Horizon server for testnet
-const server = new StellarSdk.Horizon.Server(
-  "https://horizon-testnet.stellar.org"
-);
-
-// Initalize the source account's secret key and destination account ID.
-// The source account is the one that will send the payment, and the destination account
-// is the one that will receive the payment.
-
-const sourceKeys = StellarSdk.Keypair.fromSecret(
-  "SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4"
-);
-const destinationId =
-  "GA2C5RFPE6GCKMY3US5PAB6UZLKIGSPIUKSLRB6Q723BM2OARMDUYEJ5";
-
-// First, check to make sure that the destination account exists.
-try {
-  await server.loadAccount(destinationId);
-} catch (error) {
-  console.error("Error checking destination account:", error);
-  throw error;
-}
+// send_payment.js
+
+/*
+Pre-requisites:
+1. [nvm](https://github.com/nvm-sh/nvm) utility installed to manage node version.
+2. setup local example project:
+  nvm install 20
+  nvm use 20
+  mkdir stellar-payments
+  cd stellar-payments
+  npm init -y
+  npm install --save @stellar/stellar-sdk 
+*/
```

Comment:
This is part of SDK installation instructions, maybe unnecessary here? No strong opinion.